src/20241007_Multi_GHOStewardV2Upgrade/AaveV3Arbitrum_GHOStewardV2Upgrade_20241007.t.sol

```diff
@@ -0,0 +1,415 @@
+    uint32 newBaseVariableBorrowRate = currentRates.baseVariableBorrowRate + 1;
+    uint32 newVariableRateSlope1 = currentRates.variableRateSlope1 - 1;
+    uint32 newVariableRateSlope2 = currentRates.variableRateSlope2 - 1;
+
+    vm.startPrank(RISK_COUNCIL);
+    IGhoAaveSteward(proposal.GHO_AAVE_STEWARD()).updateGhoBorrowRate(
+      newOptimalUsageRatio,
+      newBaseVariableBorrowRate,
+      newVariableRateSlope1,
+      newVariableRateSlope2
+    );
+    vm.stopPrank();
+
+    vm.clearMockedCalls();
+
+    assertEq(_getOptimalUsageRatio(), newOptimalUsageRatio);
+    assertEq(_getBaseVariableBorrowRate(), newBaseVariableBorrowRate);
+    assertEq(_getVariableRateSlope1(), newVariableRateSlope1);
+    assertEq(_getVariableRateSlope2(), newVariableRateSlope2);
+  }
+
+  function test_ghoBucketSteward_updateFacilitatorBucketCapacity() public {
+    executePayload(vm, address(proposal));
+
+    (uint256 currentBucketCapacity, ) = IGhoToken(AaveV3ArbitrumAssets.GHO_UNDERLYING)
+      .getFacilitatorBucket(MiscArbitrum.GHO_CCIP_TOKEN_POOL);
+    vm.startPrank(RISK_COUNCIL);
+    uint128 newBucketCapacity = uint128(currentBucketCapacity) + 1;
+    IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).updateFacilitatorBucketCapacity(
+      MiscArbitrum.GHO_CCIP_TOKEN_POOL,
+      newBucketCapacity
+    );
+    (uint256 bucketCapacity, ) = IGhoToken(AaveV3ArbitrumAssets.GHO_UNDERLYING)
+      .getFacilitatorBucket(MiscArbitrum.GHO_CCIP_TOKEN_POOL);
+    assertEq(bucketCapacity, newBucketCapacity);
+  }
+
+  function test_ghoBucketSteward_setControlledFacilitator() public {
+    executePayload(vm, address(proposal));
+
+    address[] memory newGsmList = new address[](1);
+    address gho_gsm_4626 = makeAddr('gho_gsm_4626');
+    newGsmList[0] = gho_gsm_4626;
+
+    vm.startPrank(GovernanceV3Arbitrum.EXECUTOR_LVL_1);
+    IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).setControlledFacilitator(newGsmList, true);
+    assertTrue(
+      IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).isControlledFacilitator(gho_gsm_4626)
+    );
+
+    IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).setControlledFacilitator(newGsmList, false);
+    assertFalse(
+      IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).isControlledFacilitator(gho_gsm_4626)
+    );
+  }
+
+  function test_ghoCcipSteward_updateRateLimit() public {
+    executePayload(vm, address(proposal));
+
+    RateLimiter.TokenBucket memory outboundConfig = IUpgradeableLockReleaseTokenPool(
+      MiscArbitrum.GHO_CCIP_TOKEN_POOL
+    ).getCurrentOutboundRateLimiterState(remoteChainSelector);
+    RateLimiter.TokenBucket memory inboundConfig = IUpgradeableLockReleaseTokenPool(
+      MiscArbitrum.GHO_CCIP_TOKEN_POOL
+    ).getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    RateLimiter.Config memory newOutboundConfig = RateLimiter.Config({
+      isEnabled: outboundConfig.isEnabled,
+      capacity: outboundConfig.capacity + 10000,
```

Comment:
Small nit, but to me I think it's clearer that we change this to `+1` and put the comment back that we can't change it even by 1, because 100% of 0 is 0.